### PR TITLE
Increase RHEL8 STIG Coverage

### PR DESCRIPTION
**Pull request by Mab879**
#### Description:

Fix STIGIDs and add prodtypes to getting close to 100% coverage for RHEL 8.3+

The reports will not state 100% as the STIG below are not applicable to RHEL 8.2+ and since we cannot have two STIG IDs they will be counted as missing.
* [RHEL-08-020010](https://stigaview.com/products/rhel8/v1r9/RHEL-08-020010/)
* [RHEL-08-020012](https://stigaview.com/products/rhel8/v1r9/RHEL-08-020012/)
* [RHEL-08-020014](https://stigaview.com/products/rhel8/v1r9/RHEL-08-020014/)
* [RHEL-08-020016](https://stigaview.com/products/rhel8/v1r10/RHEL-08-020016/)
* [RHEL-08-020018](https://stigaview.com/products/rhel8/v1r10/RHEL-08-020018/)
* [RHEL-08-020020](https://stigaview.com/products/rhel8/v1r10/RHEL-08-020020/)
* [RHEL-08-020020](https://stigaview.com/products/rhel8/v1r10/RHEL-08-020022/)
* [RHEL-08-020102](https://stigaview.com/products/rhel8/v1r10/RHEL-08-020102)
* [RHEL-08-020103](https://stigaview.com/products/rhel8/v1r10/RHEL-08-020103)

[RHEL-08-010471](https://stigaview.com/products/rhel8/v1r10/RHEL-08-010471/) is covered by `no_empty_passwords` due to the fact we cannot assign two STIGs on one rule this item will be marked as missing.

[RHEL-08-010471](https://stigaview.com/products/rhel8/v1r9/RHEL-08-010471) is not covered as rngd on 8.4+ causes issues when not in FIPS mode. See #10153 for mroe details.

#### Rationale:

Better coverage for RHEL 8 STIG.

#### Review Hints:

The `table-rhel8-stig.html` ([from master](https://complianceascode.github.io/content-pages/tables/table-rhel8-stig.html) file is useful in review.)

### CODE REVIEW

File: tests/data/profile_stability/rhel8/stig_gui.profile [hunk 1]
```diff
@@ -490,4 +491,5 @@ cpe_names: !!set {}
 platform: null
 filter_rules: ''
 title: DISA STIG with GUI for Red Hat Enterprise Linux 8
+definition_location: /home/mburket/code/ComplianceAsCode/content/products/rhel8/profiles/stig_gui.profile
```
Comment (marcusburghardt) [inline tests/data/profile_stability/rhel8/stig_gui.profile]: If you want, you can edit this commit by removing this line and so delete the next commit.

Comment (Mab879) [reply tests/data/profile_stability/rhel8/stig_gui.profile]: Done.

File: linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_minlen_login_defs/rule.yml [hunk 2]
```diff
@@ -47,6 +47,7 @@ references:
     nist-csf: PR.AC-1,PR.AC-6,PR.AC-7
     srg: SRG-OS-000078-GPOS-00046
     stigid@ol8: OL08-00-020231
+    stigid@rhel8: RHEL-08-020231
```
Comment (marcusburghardt) [inline linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_minlen_login_defs/rule.yml:50]: I couldn't find the `accounts_password_minlen_login_defs` rule or the `RHEL-08-020231` reference in the STIG profile file. Could you confirm the change, please?

**Review by marcusburghardt — CHANGES_REQUESTED** [review]

**Comment by marcusburghardt** [comment]
Unfortunately the `service_rngd_enabled` rule is failing in `testing-farm:centos-stream-8-x86_64`. We need to investigate it.

**Comment by Mab879** [comment]
> Unfortunately the `service_rngd_enabled` rule is failing in `testing-farm:centos-stream-8-x86_64`. We need to investigate it.

Per #10153 I have removed that rule again.

**Review by marcusburghardt — APPROVED** [review]
Thanks

**Pull request merged**